packages/dds/tree/src/test/simple-tree/api/schemaFactory.spec.ts

```diff
@@ -599,6 +637,24 @@ describe("schemaFactory", () => {
 			class NamedMap extends factory.map("name", factory.number) {}
 			const namedInstance = new NamedMap(new Map([["x", 5]]));
 		});
+
+		it("Node schema metadata", () => {
+			const factory = new SchemaFactory("");
+
+			class Foo extends factory.map("Foo", factory.number, {
+				metadata: { description: "A map containing numbers", custom: { baz: true } },
+			}) {}
+
+			assert.equal(Foo.metadata?.description, "A map containing numbers");
+			assert.equal(Foo.metadata?.custom?.baz, true);
+
+			const foo = hydrate(Foo, { a: 1, b: 2, c: 3 });
+			const schema = Tree.schema(foo);
```

Comment:
Tree.schema literally returns the same object you tested above, so this part of all these tests seems like it just an odd way to redundantly test Tree.schema. I'd suggest removing these parts,